Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM pytorch/pytorch:2.3.0-cuda12.1-cudnn8-runtime
```

Comment:
please, add also label to link container image to repo 

https://github.com/neuro-inc/private-gpt/blob/apolo/Dockerfile.external#L22